.github/workflows/release.yml

```diff
@@ -6,8 +6,8 @@ on:
 env:
   CARGO_TERM_COLOR: always
   IMAGE_NAME: plerkle-test-validator
-  RUST_VERSION: 1.73.0
-  SOLANA_VERSION_STABLE: v1.17.20
+  RUST_VERSION: 1.75.0
+  SOLANA_VERSION_STABLE: v1.18.20
```

Comment:
Actually could we use consistent Solana version here as what is in crate?  The reason being is that it generates a Docker image with this version.

Reply:
fixed